### PR TITLE
[ENG-5289] - Add Shine icon 🚀

### PR DESCRIPTION
**Pull request by cbxalves**
[JIRA Issue](https://gympass.atlassian.net/browse/ENG-5289)

## Description 📄

- Add new Shine icon

## Platforms 📲

- [x] Web
- [x] Mobile

## Type of change 🔍

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? 🧪

[Enter the tips to test this PR]

- [ ] Unit Test
- [ ] Snapshot Test

## Checklist: 🔍

- [x] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [ ] Layout matches design prototype: [FIGMA](https://figma.com/file/YOUR_LINK)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Screenshots 📸

N/A

### CODE REVIEW

**Review by diogoverde — APPROVED** [review]

File: packages/icons/src/svg/shine.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ericcleao) [inline packages/icons/src/svg/shine.svg:1]: Look at the current icon patterns from the `svg` folder, every icon is using the `12px` version of the icon to be defined, could we update it to use the same?

File: packages/icons/src/ui.js [hunk 2]
```diff
@@ -133,6 +133,7 @@ import ShareAndroid from './svg/share_android.svg';
 import ShareIos from './svg/share_ios.svg';
 import ShieldCheck from './svg/shield_check.svg';
 import ShieldSecure from './svg/shield_secure.svg';
+import Shine from './svg/shine.svg';
```
Comment (ericcleao) [inline packages/icons/src/ui.js:136]: On the `Yoga` project this icon is defined under the `Filled` icons, even if we do not have a `not filled` version now it is probably better to create it as `Filled` and let the name `Shine` for the not filled one, WDYT?

```suggestion
import ShineFilled from './svg/shine_filled.svg';
```

File: packages/icons/src/svg/shine.svg (hunk 1 above)
Comment (cbxalves) [reply packages/icons/src/svg/shine.svg:1]: Done! Thanks for noticing 👍

File: packages/icons/src/ui.js (hunk 2 above)
Comment (cbxalves) [reply packages/icons/src/ui.js]: Done!

**Review by ericcleao — APPROVED** [review]

**Pull request merged**